src/main/java/chess/domain/position/Column.java

```diff
@@ -0,0 +1,71 @@
+package chess.domain.position;
+
+import chess.domain.chessPiece.Team;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toMap;
+
+public class Column {
+    private final int value;
+    private static final Map<Integer, Column> CACHE = IntStream.rangeClosed(0, 7)
+            .boxed()
+            .collect(toMap(Function.identity(), Column::new));
```

Comment:
```suggestion
    private static final Map<Integer, Column> CACHE = IntStream.rangeClosed(0, 7)
            .boxed()
            .collect(toMap(Function.identity(), Column::new));
    private final int value;
```
자바 컨벤션상 static이 위에 배치되어야해요.